src/application.rs

```diff
@@ -159,6 +159,11 @@ impl CoLink {
 
     pub async fn create_entry(&self, key_name: &str, payload: &[u8]) -> Result<String, Error> {
         let mut client = self._grpc_connect(&self.core_addr).await?;
+        #[cfg(feature = "magic_mount")]
```

Comment:
It's nice that we have this conditional compilation. However, when a user (who happens to have disabled the feature) accidentally uses this feature, there will be an unexpected error.
So instead I suggest the following: always check if there is a dollar sign during the runtime. If the user disables the feature but still triggered the "contains $" branch, throw the error. This might add some overhead but string checking should be not that time-consuming.